lib/utils/utils.ts

```diff
@@ -115,6 +115,18 @@ export const derivePairId = (pair: Pair) => {
   return `${pair.baseCurrency}/${pair.quoteCurrency}`;
 };
 
+/**
+ * Convert ticker symbole into pair.
+ * @param { string } pair - example: btc/ltc
+ */
+export const getPairId = (pair: string): Pair => {
```

Comment:
This functionality is used a couple other places in the code base, and I think it's most concisely expressed simply with:

```javascript
const [baseCurrency, quoteCurrency] = pairId.split('/');
```

So I'd prefer to use that line instead of this function for consistency and conciseness.